Use Cases/SDPi/Temp from Repo/tf1-ch-c-use-case-stad.adoc

```diff
@@ -0,0 +1,97 @@
+<<<<<<< HEAD
+// = Use Case Feature: Synchronized Time Across Devices (STAD)
+
+[sdpi_offset=2]
+=== Use Case Feature {var_use_case_id}: Synchronized Time Across Devices (STAD)
+
+==== Narrative:
+Nurse Jean attaches a ventilator to the SDC based network in the ICU.  It automatically synchronizes its clock with the clocks of the other devices on the network.
+
+=======
+// = Use Case Feature: Standalone OR Dashboard (SORD)
+
+
+NOTE:  This file contains the use case content for SCAD.  See the Word document for the baseline.  Gherkin plugin may be utilized.
+
+// C.2
+// [appendix#vol1_appendix_c_dpi_use_cases,sdpi_offset=C]
+[sdpi_offset=2]
+=== Use Case Feature: SYNCHRONIZED TIME ACROSS DEVICES (SCAD)
+
+// C.2.1
+==== Narrative:
+Nurse Jean attaches a ventilator to the SDC based network in the ICU.  It automatically synchronizes its clock with the clocks of the other devices on the network.
+
+// C.2.2
+>>>>>>> 1d40947356fc3f12820a28dc1023dc3e0aecc594
+==== Technical View
+
+##TODO:  Add graphic here##
+
+<<<<<<< HEAD
+=======
+// C.2.3
+>>>>>>> 1d40947356fc3f12820a28dc1023dc3e0aecc594
+==== Technical Pre-Conditions
+
+*Given* All devices communicate using SDC
+
+*And* NTP service on the SDC network
+
+<<<<<<< HEAD
+==== Scenarios
+
+===== Scenario: STAD {var_use_case_id}.1 - Device is connected to network with NTP service
+=======
+// C.2.4
+==== Scenarios
+
+===== Scenario: STAD 1.1 - Device is connected to network with NTP service
+>>>>>>> 1d40947356fc3f12820a28dc1023dc3e0aecc594
+
+*Given* Device has detected at least one NTP service
+
+*When* The NTP service is operational
+
+*Then* The device will synchronize its time with the NTP service time
+
+<<<<<<< HEAD
```

Comment:
You have diff annotations in your source. This needs to be fixed.

Reply:
This was fixed in an update to the branch after it was originally created.  Ensure that you are working with the latest branch sources.  

NOTE:  For this file, at some point during the PR process, it was significantly out-of-sync (as indicated in this comment).  HOW that happened remains a mystery but as far as I can tell, it was reconciled the same day the PR was created.

Reply:
I'm working with this PR as a source. Is it by any means possible that you have changes comitted locally which have not been pushed to github yet?